### PR TITLE
Add integration test for sign up flow

### PR DESCRIPTION
**Pull request by kachkaev**

### CODE REVIEW

File: site/tests/integration/sign-up-flow.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  await expect(
+    page.locator('text=This user has already been taken"'),
+  ).not.toBeVisible();
+
+  await preferredNameInput.fill("Alice");
+
+  // @todo Remove when Safari bug is fixed
+  // (main page scrolls down when modal is closed, navbar disappears)
+  test.skip(
```
Comment (teenoh) [inline site/tests/integration/sign-up-flow.ts:83]: I assume test.skip should be used if we want to skip the entire test as opposed to a section. I think it's better to use a conditional to handle the remaining parts of the test

```
if (browserName !== "webkit")
{
  ...
}
```

Comment (teenoh) [reply site/tests/integration/sign-up-flow.ts:83]: This can be addressed in a follow up

**Review by teenoh — APPROVED** [review]

File: site/tests/integration/sign-up-flow.ts (hunk 1 above)
Comment (kachkaev) [reply site/tests/integration/sign-up-flow.ts:83]: WDYT of using `test.fixme` instead?

Comment (teenoh) [reply site/tests/integration/sign-up-flow.ts:83]: Yeah we should definitely use that

**Pull request merged**